ds_judgements_public_ui/sass/includes/_search.scss

```diff
@@ -48,6 +48,33 @@
     }
   }
 
+  &__more-options-beta {
+    margin-top: calc($spacer-unit / 2);
+
+    a {
+      color: $color-dark-blue;
+      text-decoration: none;
+      border: 0;
+      font-size: 1.1rem;
+      position: relative;
+      padding: 0 0 calc($spacer-unit / 2);
+      border-bottom: 0.125rem solid $color-dark-blue;
```

Comment:
On the design, the border style looks slightly different to this. Not sure if we wanted to follow that or not?

Reply:
@jlhdxw thanks for spotting took out the padding and updated border to 1px